src/elements/icon/icon.visual.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { html } from 'lit';
```

Comment:
looking at icon.scss, tests which could be added are color mutations, and testing color-immutable. And if it's possible a test with "data-empty" (no icon loaded). If it's too time consuming we can save it for another improvement ;)

Reply:
I can easily add a test with color=red and a test with no icon name; 
for the color-immutable in the call stack from 'loadSvgIcon' to 'validateContent' i see no way to do it programmatically